packages/lodestar/test/unit/network/peers/utils/enrSubnets.test.ts

```diff
@@ -28,4 +28,88 @@ describe("ENR syncnets", () => {
       ).to.deep.equal(bools);
     });
   }
+
+  // ATTESTATION_SUBNET_COUNT = 64 is too much for unit test, make it 16
+  const ATTESTATION_SUBNET_COUNT = 16;
+  const attnetTestCases: {bytes: string; bools: boolean[]}[] = [
+    {
+      bytes: "1111",
+      bools: [
+        true,
+        false,
+        false,
+        false,
+        true,
+        false,
+        false,
+        false,
+        true,
+        false,
+        false,
+        false,
+        true,
+        false,
+        false,
+        false,
+      ],
+    },
+    {
+      bytes: "2222",
+      bools: [
+        false,
+        true,
+        false,
+        false,
+        false,
+        true,
+        false,
+        false,
+        false,
+        true,
+        false,
+        false,
+        false,
+        true,
+        false,
+        false,
+      ],
+    },
+    {
+      bytes: "3333",
+      bools: [true, true, false, false, true, true, false, false, true, true, false, false, true, true, false, false],
+    },
+    {
+      bytes: "4444",
+      bools: [
+        false,
+        false,
+        true,
+        false,
+        false,
+        false,
+        true,
+        false,
+        false,
+        false,
+        true,
+        false,
+        false,
+        false,
+        true,
+        false,
+      ],
+    },
+    {
+      bytes: "ffff",
+      bools: [true, true, true, true, true, true, true, true, true, true, true, true, true, true, true, true],
+    },
+  ];
+  for (const {bytes, bools} of attnetTestCases) {
+    const bytesBuf = Buffer.from(bytes, "hex");
+    it(`Deserialize attnet ${bytes}`, () => {
+      expect(
+        deserializeEnrSubnets(bytesBuf, ATTESTATION_SUBNET_COUNT).slice(0, ATTESTATION_SUBNET_COUNT)
```

Comment:
seeing this has already been merged, this is more of a question...is the `slice` needed in the test? I removed it, that is:

```
expect(deserializeEnrSubnets(bytesBuf, ATTESTATION_SUBNET_COUNT)).to.deep.equal(bools)
```
and the tests still passed alright.

Reply:
yaah it's not really needed, it's only needed for sync committee, the test above I guess. However the test overall still serves our purpose.